src/eig_problem/MacierzDoZagadnienia.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, ilosc_wektorow, skad_wspolczynnik, typ_pola_wymiany):
             self.slownik_dlugosc_wymiany = wspolczynniki[1]
             self.slownik_magnetyzacja = wspolczynniki[0]
         self.lista_wektorow = WektorySieciOdwrotnej(self.a, self.b, ilosc_wektorow).lista_wektorow('min')
-        self.pole_wymiany_dll = CDLL('./pole_wymiany_c/pole_wymiany_ii.so')
+        self.pole_wymiany_dll = CDLL('src/eig_problem/pole_wymiany_c/pole_wymiany_ii.so')
```

Comment:
Simple way to make script work. It's executed from the main directory, so this path should be relative to it.